apis/k8ssandra/v1alpha1/k8ssandracluster_webhook.go

```diff
@@ -144,6 +147,56 @@ func (r *K8ssandraCluster) ValidateUpdate(old runtime.Object) error {
 	return nil
 }
 
+func validateNoRackRenamed(oldCassandra, newCassandra *CassandraClusterTemplate) error {
```

Comment:
I would probably give a custom type the error you're returning, but this is a suggestion not a requirement.